spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -892,18 +895,20 @@ exposed method from unauthorized access. The specific security details are not s
 ==== Eventual compensations
 
 [[async-compensators]]
-If the resource cannot perform a compensation activity
-immediately the `@Compensate` method SHOULD do one or more of the following:
+If a resource cannot perform a compensation activity
+immediately the `@Compensate` method MUST report that the activity
+is still in progress using one of the following options:
 
-- return `ParticipantStatus.Compensating`
-- return a `202 Accepted` HTTP status code
-- mark the method as a JAX-RS asynchronous method (using the
-  `javax.ws.rs.container.Suspended` annotation).
+- Return a `CompletionStage` or mark the method as asynchronous
+  (using the `javax.ws.rs.container.Suspended` annotation). The future must
+  deliver the final status when the stage completes. Please refer to the section
+  <<reactive-support,about reactive support>> for more details.
```

Comment:
Should we also mention what should happen when CompletionStage delivers Compensating?

Reply:
I added some text